src/client/src/Types/Permissions.ts

```diff
@@ -0,0 +1,12 @@
+export enum Permissions {
+    CANGETSITES = "site:get-config",
```

Comment:
Block capitals is difficult for the human brain to decipher, I would remove the "can" from the start and make these Pascal case for readability